invenio_rdm_records/resources/serializers/datacite/__init__.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""DataCite Serializers for Invenio RDM Records."""
+
+from flask_resources.serializers import SerializerMixin
+
+from .schema import RDMDataCite43Schema
+
+
+class BaseRDMDataCiteJSONSerializer(SerializerMixin):
```

Comment:
You can use https://github.com/inveniosoftware/flask-resources/blob/master/flask_resources/serializers/json.py#L75 i think

Reply:
Totally right, thanks!